searx/static/themes/simple/src/js/main/results.js

```diff
@@ -16,6 +16,8 @@
           }
         ));
 
+    d.querySelector('#search_url button#copy_url').style.display = "block";
```

Comment:
@BernieHuang2008 @mrpaulblack this line is responsible for https://github.com/searxng/searxng/issues/3003 .. can you please have a look / thanks!